src/torchmetrics/classification/accuracy.py

```diff
@@ -325,13 +324,11 @@ def compute(self) -> Tensor:
 
 
 class Accuracy(StatScores):
-    r"""
-    .. note::
-        From v0.10 an `'binary_*'`, `'multiclass_*', `'multilabel_*'` version now exist of each classification
-        metric. Moving forward we recommend using these versions. This base metric will still work as it did
-        prior to v0.10 until v0.11. From v0.11 the `task` argument introduced in this metric will be required
-        and the general order of arguments may change, such that this metric will just function as an single
-        entrypoint to calling the three specialized versions.
+    r""".. note:: From v0.10 an `'binary_*'`, `'multiclass_*', `'multilabel_*'` version now exist of each
```

Comment:
I am not sure if all these will be rendered correctly, so we probably need to add online everywhere...